.github/CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 <h3>New features since last release</h3>
 
+* Added the `DoubleExcitation` four-qubit operation, which is useful for quantum
+  chemistry applications. [(#1123)](https://github.com/PennyLaneAI/pennylane/pull/1123)
+
+  It can be used to perform an SO(2) rotation in the subspace 
+  spanned by the states :math:`|1100\rangle` and :math:`|0011\rangle`. 
+  For example, the following circuit performs the transformation
+  :math:`|1100\rangle\rightarrow \cos(\phi/2)|1100\rangle - \sin(\phi/2)|0011\rangle`:   
+
+  ```python
+
+  @qml.qnode(dev)
+  def circuit(phi):
+      qml.PauliX(wires=0)
+      qml.PauliX(wires=1)
+      qml.SingleExcitation(phi, wires=[0, 1, 2, 3])
+  ```
+
+  The `DoubleExcitation` operation supports analytical gradients on hardware using only four 
```

Comment:
Have to say, not quite sure if there's a huge difference, suggestion is based on usage in [Evaluating analytic gradients on quantum hardware](https://arxiv.org/pdf/1811.11184.pdf).
```suggestion
  The `DoubleExcitation` operation supports analytic gradients on hardware using only four 
```